### DISCUSSION TITLE
Getting the Nodata in Grafana

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1634] [Q&A] [answered]
Hi,
While I'm running the queries by giving the metrics  as up{name="frontend-ansible". I'm getting as No data
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/cc2e2de7-3752-4b6c-b712-e72f41685ab4)
Grafana URL:   52.90.212.14:3000
Below is the error status I'm getting
Status: 500. Message: bad_data: invalid parameter "query": 1:8: parse error: unexpected "="

**Comment by sreedharm07** [comment]
@omswetha 
 Only up is giving data ?

**Reply by sreedharm07** [reply]
You are giving wrong input 
Please guve correct quarie

**Reply by omdevops99** [reply]
Yes, Only UP is giving the data

**Reply by sreedharm07** [reply]
Then your input is wrong give correct quary

**Reply by omdevops99** [reply]
is this the wrong query  up{name="frontend-ansible"}

**Reply by sreedharm07** [reply]
I think yes 
I dont remember the quary but 
This is the wrong one may be please check once

**Reply by omdevops99** [reply]
I'm giving the same which raghu has given but still the same

**Reply by sreedharm07** [reply, marked as answer]
i seen in your prometheus server there is nothing there with name  so its not comming 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3ba6b2f8-801d-45d1-af4a-f6ba5179a7e1)

**Reply by omdevops99** [reply]
Okay, Thanks.
